plugins/hydra_ax/example/banana.py

```diff
@@ -8,12 +8,13 @@
 
 @hydra.main(config_path="conf/config.yaml")
 def banana(cfg):
+    print(cfg)
     a = cfg.banana.a
     b = cfg.banana.b
     x = cfg.banana.x
     y = cfg.banana.y
-    z = (a - x) ** 2 + b * (y - x ** 2) ** 2
-    log.info("Banana_a_{}_b_{}({}, {})={}".format(a, b, x, y, z))
+    z = (a - x) ** 2 + b * ((y - x ** 2) ** 2)
+    log.info("Banana_Function_a={}_b={}(x={}, y={})={}".format(a, b, x, y, z))
```

Comment:
Feel free to use Python 3 fstring here, this plugin depends on Python 3 through Ax anyway.